packages/ios_platform_images/pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: ios_platform_images
 description: A plugin to share images between Flutter and iOS in add-to-app setups.
 repository: https://github.com/flutter/plugins/tree/main/packages/ios_platform_images
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+ios_platform_images%22
-version: 0.2.1
+version: 0.2.2
 
 environment:
-  sdk: ">=2.14.0 <3.0.0"
+  sdk: '>=2.18.0 <3.0.0'
```

Comment:
This should have been bumped with 0.2.1 when updated to 3.3.

Reply:
We don't generally bother to keep them in sync. I've thought about having tooling enforce expected pairs, but I haven't come up with any benefit from doing so.

So the current approach is that we mostly only update each one when there is a specific reason for that constraint. Usually it's just the Flutter constraint, but things like NNBD or new Dart features trigger an `sdk` update. Or when people feel like it, like here :)